amqp-salesforce/src/main/resources/application.properties

```diff
@@ -39,4 +39,9 @@ camel.component.salesforce.clientSecret=1039611643161946846
 #camel.component.salesforce.refreshToken=5Aep861HDR3iASSXIX6hI7M1qMWSCs1Ym57WUH1ftjE7RvnM7MvnAXx9EZaw_HIoNCKktNsuhx.xwjLThofuJH4
 ## you can also use:
 #camel.component.salesforce.userName=<Salesforce username>
-#camel.component.salesforce.password=<Salesforce password>
\ No newline at end of file
+#camel.component.salesforce.password=<Salesforce password>
+
+# expose actuator endpoint via HTTP
+management.endpoints.web.exposure.include=camelroutes,metrics,prometheus,health
+# turn on actuator health check
+management.endpoint.health.enabled = true
```

Comment:
is it necessary? [management.endpoint.health.enabled = true](https://docs.spring.io/spring-boot/docs/current/reference/html/application-properties.html#application-properties.actuator.management.endpoint.health.enabled) is the default value

Reply:
I don't think it is necessary but it is not going to hurt and it has been included in several of the quickstarts.